src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmClrRuntimeInstance.cs

```diff
@@ -89,6 +91,27 @@ internal DkmClrType GetType(string typeName, params System.Type[] typeArguments)
             return null;
         }
 
+        private static IEnumerable<DkmClrModuleInstance> WithMscorlibLast(DkmClrModuleInstance[] list)
+        {
+            DkmClrModuleInstance mscorlib = null;
+            foreach (var module in list)
+            {
+                if (module.Assembly.GetReferencedAssemblies().Length == 0)
```

Comment:
>if (module.Assembly.GetReferencedAssemblies().Length == 0) [](start = 16, length = 58)

This check is not accurate. I think it is possible to have an assembly without references that is not Core Library.